formfactory/models.py

```diff
@@ -140,7 +140,7 @@ def as_form(self, data=None, files=None):
             )
 
         return factory.FormFactory(
-            data, files, prefix=self.slug, fields=self.fields.all(),
+            data, files, prefix=self.slug, field_groups=self.fieldgroups.all(),
```

Comment:
The `self.fieldgroups.all()` should probably be ordered by `"fieldgroupformthrough"`